README.md

```diff
@@ -1,3 +1,7 @@
 # learngit - covers absolute basics for everyone
 
 This git repository will cover git basic commands and workflows.
+
+# Cool Feature #1
```

Comment:
First topic should be initializing a git repository

Reply:
Taken care of this comment in my force-push.